sql-plugin/src/main/scala/org/apache/spark/sql/rapids/RapidsShuffleInternalManagerBase.scala

```diff
@@ -770,20 +831,27 @@ abstract class RapidsShuffleThreadedReaderBase[K, C](
 
     private def deserializeTask(blockState: BlockState): Unit = {
       val slot = RapidsShuffleInternalManagerBase.getNextReaderSlot
+      val tc = TaskContext.get()
```

Comment:
This is not used and is showing as a compile error.

Reply:
whoops, thanks